Frameworks/SPMySQLFramework/TimeZone.m

```diff
@@ -0,0 +1,50 @@
+//
+//  TimeZone.m
+//  SPMySQL.framework
+//
+//  Created by Robin Kunde on 8/19/20.
+//
+
+#import "TimeZone.h"
+#import "SPMySQLStringAdditions.h"
+
+@implementation SPMySQLConnection (TimeZone)
+
+#pragma mark -
+#pragma mark Current connection time zone information
+
+/**
+ * Returns the time zone identifier in use by the connection.
+ */
+- (NSString *)timeZoneIdentifier
+{
+    return [NSString stringWithString:timeZoneIdentifier];
```

Comment:
This seems like a ... cycle? What is `timeZoneIdentifier` - isn't it getter of the same property we are in?

Reply:
No, that's the property itself. This getter has to be invoked as a method call, either via dot syntax (`self.timeZoneIdentifier`) or regular (`[self timeZoneIdentifier]`). I based this on the other `SPMySQLConnection` categories, specifically `Encoding.h/m`.